packages/app-headless-cms-common/src/types/index.ts

```diff
@@ -17,6 +17,7 @@ import {
 import { CmsModel, CmsModelField } from "./model";
 import { CmsIdentity } from "~/types/shared";
 import type { SourceType } from "dnd-core";
+import { IconDto } from "@webiny/admin-ui";
```

Comment:
Naming.... constant pain :( 

It's just hard to know what's this `IconDto` related to. 

Like... if it was maybe `IconPickerIconDto`, that would be more understandable. But then ofc that's a looooong name. 

I'm not saying we do anything about it now, but we might want to note this down somewhere / see how we want to deal with these things.